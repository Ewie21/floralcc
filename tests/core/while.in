int n = '8';
while (n != '0') {
    put(n);
    n -= 1;
}
put(n);
put('\n');
int after_test = '5';
put(after_test);
