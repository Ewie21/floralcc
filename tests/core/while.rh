int n = '8';
while (n != '0') {
    put(n);
    n -= 1;
}
