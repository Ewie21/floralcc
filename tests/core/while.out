876543210
5
